test-utils/describe-rule.js

```diff
@@ -1,19 +1,42 @@
 const getRulesMarkdownData = require('../utils/get-rules-markdown-data')
 
+/**
+ * Get ids of rules that are of given type
+ * @param {Array<Object>} rules list of rules
+ * @param {String} ruleType type of rule
+ */
+const getRuleIdsOfRuleType = (rules, ruleType) => {
```

Comment:
I'd rather split the two steps (filtering and getting Ids) in order to have simpler code, easier to understand and maintain.

Something like:
```javascript
return rules
  .filter(rule => rule.frontmatter.rule_type === ruleType)
  .map(rule => rule.frontmatter.id)
```

Reply:
2 iterations vs 1 iteration ;). Given we are only iterating and adding to an array think it is easy to follow. Can change it to a for.. loop instead of reduce if you prefer.

Reply:
Given that this is not exactly time-critical, I'd go for readability over performance without hesitation.